src/utils/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const getModalAppendTo = (): HTMLElement =>
```

Comment:
This doesn't belong to this repo, it's an app-services-ui concern knowing there to put modals